### PR TITLE
🐛Apester Media(extension): Fixed resizing

### PR DESCRIPTION
**Pull request by OmriKeret**
# New media types requires changing of media's width and not just media's height.

- fix: resize now changes media's width and not just media's height

### CODE REVIEW

**Comment by OmriKeret** [comment]
Hi @aghassemi, We've developed a fix for our resizing method.
Could you please review it?

Thanks!

File: extensions/amp-apester-media/0.1/amp-apester-media.js [hunk 1]
```diff
@@ -426,6 +436,15 @@ class AmpApesterMedia extends AMP.BaseElement {
...
   }
 
+  /**
+   * Changes the apester-media size.
+   * @param {number} height
+   * @param {number} width
+   */
+  changeUnitSize(height, width) {
+    this./*OK*/ attemptChangeHeight(height);
+    setStyle(this.element, 'max-width', width);
```
Comment (aghassemi) [inline extensions/amp-apester-media/0.1/amp-apester-media.js]: how would this work for `layout=responsive` which is supposed to keep width at 100% of parent?

Comment (OmriKeret) [reply extensions/amp-apester-media/0.1/amp-apester-media.js]: Hi @aghassemi, Thanks for your response!
I took a second look at the AMP layout options, I think that our current supported layouts are not correctly put.

I think it's "intrinsic" (and intrinsic alone) as the Apester-Media component size is based on the parent container and aspect ratio of the unit type - It differs from AMP definition of unit's width:height ratio, so I would need your input on that to make sure I'm right.
WDYT?

Thanks!

Comment (aghassemi) [reply extensions/amp-apester-media/0.1/amp-apester-media.js]: @OmriKeret There isn't much difference between `intrinsic` and `responsive`. If vast majority of cases `intrinsic` and `responsive` behave exactly the same way. Only in cases where a default size is need (e.g. parent has `float`) then `intrinsic` behaves differently.

In AMP, we don't normally resize `width`, page authors needs to pick the layout that makes sense for them.
`layout=responsive` would be `width:100%` and height would be kept at the same aspect ratio of the specified `width and height`. In this case many components resize `height` if needed and break that aspect ratio.
`layout=fixed` would be honored exactly as user as specified. If things overflow, so be it.

I just don't see a case where you need to change `width` ever.

Comment (OmriKeret) [reply extensions/amp-apester-media/0.1/amp-apester-media.js]: Hi @aghassemi, Thanks for your input!
Got you there.
I will talk with my coworker about changing the height alone in order to achieve the correct aspect ratio for the unit type.

I'm gonna close this PR until we decide on an approach.

Thanks!

**Pull request closed without merging**